discord/threads.py

```diff
@@ -106,7 +106,7 @@ class Thread(Messageable, Hashable):
         Bots and users with :attr:`~Permissions.manage_channels` or
         :attr:`~Permissions.manage_messages` bypass slowmode.
     message_count: :class:`int`
-        An approximate number of messages in this thread. This caps at 50.
+        The current number of messages in this thread.
```

Comment:
```suggestion
        An approximate number of messages in this thread.
```

Try not to oversell it.